tests/Platforms/OraclePlatformTest.php

```diff
@@ -563,10 +564,12 @@ public function testUsesSequenceEmulatedIdentityColumns(): void
 
     public function testReturnsIdentitySequenceName(): void
     {
-        self::assertSame('MYTABLE_SEQ', $this->platform->getIdentitySequenceName('mytable', 'mycolumn'));
-        self::assertSame('"mytable_SEQ"', $this->platform->getIdentitySequenceName('"mytable"', 'mycolumn'));
-        self::assertSame('MYTABLE_SEQ', $this->platform->getIdentitySequenceName('mytable', '"mycolumn"'));
-        self::assertSame('"mytable_SEQ"', $this->platform->getIdentitySequenceName('"mytable"', '"mycolumn"'));
+        self::assertSame('MYTABLE_MYCOLUMN_SEQ', $this->platform->getIdentitySequenceName('mytable', 'mycolumn'));
+        self::assertSame('"mytable_mycolumn_seq"', $this->platform->getIdentitySequenceName('"mytable"', 'mycolumn'));
+        self::assertSame('"mytable_mycolumn_seq"', $this->platform->getIdentitySequenceName('mytable', '"mycolumn"'));
+        self::assertSame('"mytable_mycolumn_seq"', $this->platform->getIdentitySequenceName('"mytable"', '"mycolumn"'));
```

Comment:
Why is it required that the schema manager generates the same sequence name for three different tables/columns. Isn't the issue that the patch is trying to solve?

Reply:
The patch solves the issue of identifiers being too long. These tests are only for shorter identifiers depending on of they are quoted or not

Reply:
If an identifier is unquoted, it should be upper-cased before concatenation with another one or a suffix, and the result must be always quoted. This way, this logic will be the same as the logic on the server side.

Reply:
Done. Names will now look like:
```
mytable_MYCOLUMN_seq
MYTABLE_mycolumn_seq
```